util-core/src/main/scala/com/twitter/concurrent/AsyncStream.scala

```diff
@@ -609,6 +609,7 @@ object AsyncStream {
     extends AsyncStream[A] {
     private[this] lazy val _more: AsyncStream[A] = next()
     def more(): AsyncStream[A] = _more
+    override def toString(): String = s"Cons($fa, $next)"
```

Comment:
ok fair enough. i realize it was doing toString, it just makes me squeamish. not gonna hold this up though.